docs/configuration.md

```diff
@@ -37,6 +37,7 @@ The following environment variables are available:
 | `KUBE_API_BURST`                                 | Burst to use for the Kubernetes API client. See [Config.Burst]. A value of 0 or lower will use the default from client-go, which currently is 10. Default is 0.                                                                                                                                                                                                                                                                                                                                                                                                          |
 | `KUBE_API_QPS`                                   | QPS to use for the Kubernetes API client. See [Config.QPS]. A value of 0 or lower will use the default from client-go, which currently is 5. Default is 0.                                                                                                                                                                                                                                                                                                                                                                                                               |
 | `VULNERABILITY_COUNT_LIMIT`                      | holds vulnerability count limit if vulnerability scan is enabled for the output image. If it is defined as 10, then it will output only 10 vulnerabilities sorted by severity in the buildrun status.Output. Default is 50.                                                                                                                                                                                                                                                                                                                                              |
+| `WRITABLE_HOME_DIR`                             | Specifies the path to be used as the writable home directory (HOME) for containers. By setting this environment variable, the controller will automatically mount a writable `emptyDir` volume at the specified path, which will then be used as the HOME directory for build steps. This is useful for scenarios where a writable HOME is required, such as when tools or processes need to write to the home directory. If you set `WRITABLE_HOME_DIR` to a custom path (e.g., `/my-writable-home`), ensure that this path is not used for other purposes in your container. Default is `/shared-home`. |
```

Comment:
This sounds like it would be mounted to all containers, but I am not seeing it mounted to the strategy steps, or have I missed that?